app/views/root/index.html.erb

```diff
@@ -0,0 +1 @@
+<div id="root">
```

Comment:
- [ ] I couldn't see anything in the UI. Please fix that.
![Screenshot from 2023-03-30 13-44-08](https://user-images.githubusercontent.com/26767383/228777439-4fd8e17c-fcf0-439e-a9ed-048aa57ff864.png)

Reply:
you need to run `rails s` and `yarn build --watch` in different terminals for the application to work